Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM bellsoft/liberica-openjdk-alpine:latest as builder
```

Comment:
maybe us 17.0 for tag instead of latest so we are specifically targeting a major.minor